### ISSUE TITLE
Error: Cannot read property 'attributes' of undefined

### ISSUE THREAD

**Issue by alanaasmaa** [boazpoolman/strapi-plugin-sitemap#20]
I am receiving  `Cannot read property 'attributes' of undefined`  when I click "Add" or "edit"
![image](https://user-images.githubusercontent.com/12788337/104578831-e0e83f00-5663-11eb-94d8-29c1fe24ff14.png)

Both of collections have UID field. I'm also new to strapi so I might do something wrong. Using latest version.

EDIT: Also reciving error when deleting all and pressing "add 1 by 1"
![image](https://user-images.githubusercontent.com/12788337/104580178-936cd180-5665-11eb-9b62-f6bb9f05cba3.png)

Some sc of error:
![image](https://user-images.githubusercontent.com/12788337/104580210-9f589380-5665-11eb-912a-ea7e4aeda8fb.png)
![image](https://user-images.githubusercontent.com/12788337/104580218-a2538400-5665-11eb-85f8-aca8cd03e992.png)

**Comment by boazpoolman** [comment]
Thank you for your issue submission!
If you could give me the following information I will take a look at it:
- Version of Strapi
- Version of sitemap plugin
- A JSON dump of plugin_sitemap_settings found in the core_store table of your database

**Comment by Fufuxx** [comment]
Having the same issue since upgrade to strapi `3.4.1`.  Plugin is version `^1.2.2`.

**Comment by Fufuxx** [comment]
Not sure about Json dump but that's what on the table on plugin_sitemap_settings key:
```
{"hostname":"our_host","includeHomepage":true,"excludeDrafts":true,"contentTypes":{"article":{"priority":"1","changefreq":"monthly","uidField":"friendly","area":"blog"}},"customEntries":{"blog":{"priority":0.5,"changefreq":"monthly"}}}
```

**Comment by alanaasmaa** [comment]
Strapi version: 3.4.3
Sitemap version: 1.2.3
I'm not at office currently so I can give you dump later.

**Comment by boazpoolman** [comment]
This will suffice, thank you.
I will take a look at it.

**Comment by boazpoolman** [comment]
I've created a PR.
Can you confirm that it fixes the issue?
You can test it in your project like so:

`yarn add boazpoolman/strapi-plugin-sitemap#pull/21/head`
`npm install boazpoolman/strapi-plugin-sitemap#pull/21/head`

**Comment by Fufuxx** [comment]
Yep works for me locally ! Thanks a lot for looking into it, and for this cool plugin. Really handy to get the Sitemap sorted :)

**Comment by boazpoolman** [comment]
No worries!
The fix is released in version [1.2.4](https://www.npmjs.com/package/strapi-plugin-sitemap/v/1.2.4)
